src/domain/occurrence/__tests__/CreateOccurrencePage.test.tsx

```diff
@@ -0,0 +1,275 @@
+import { MockedResponse } from '@apollo/react-testing';
+import { format } from 'date-fns';
+import { advanceTo, clear } from 'jest-date-mock';
+import React from 'react';
+
+import {
+  EventDocument,
+  MyProfileDocument,
+  OccurrenceNode,
+  PlaceDocument,
+  VenueDocument,
+} from '../../../generated/graphql';
+import * as graphqlFns from '../../../generated/graphql';
+import {
+  fakeEvent,
+  fakeLocalizedObject,
+  fakeLocation,
+  fakeOccurrences,
+  fakeOrganisations,
+  fakePerson,
+  fakePEvent,
+  fakeVenue,
+} from '../../../utils/mockDataUtils';
+import {
+  renderWithRoute,
+  screen,
+  userEvent,
+  waitFor,
+} from '../../../utils/testUtils';
+import { ROUTES } from '../../app/routes/constants';
+import CreateOccurrencePage from '../CreateOccurrencePage';
+
+const eventName = 'Testitapahtuma';
+const eventMock = fakeEvent({
+  name: fakeLocalizedObject(eventName),
+  startTime: '2020-07-13T05:51:05.761000Z',
+});
+const placeMock = fakeLocation({
+  streetAddress: fakeLocalizedObject('Testikatu'),
+});
+const venueMock = fakeVenue();
+
+advanceTo(new Date(2020, 7, 2));
+const fakeOccurrenceOverrides: Partial<OccurrenceNode>[] = [
+  {
+    startTime: '2020-08-03T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-04T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-05T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-06T09:00:00+00:00',
+  },
+  {
+    startTime: '2020-08-07T09:00:00+00:00',
+  },
+];
+
+const occurrenceFormData = {
+  date: '13.08.2020',
+  startsAt: '12:00',
+  endsAt: '13:00',
+  amountOfSeats: '30',
+  minGroupSize: '10',
+  maxGroupSize: '20',
+};
+
+const apolloMocks: MockedResponse[] = [
+  {
+    request: {
+      query: MyProfileDocument,
+      variables: {},
+    },
+    result: {
+      data: { myProfile: fakePerson({ organisations: fakeOrganisations() }) },
+    },
+  },
+  {
+    request: {
+      query: EventDocument,
+      variables: {
+        id: eventMock.id,
+        include: ['location'],
+      },
+    },
+    result: {
+      data: {
+        event: {
+          ...eventMock,
+          pEvent: fakePEvent({
+            occurrences: fakeOccurrences(5, fakeOccurrenceOverrides),
```

Comment:
This is really minor thing. Maybe this could be
`occurrences: fakeOccurrences(fakeOccurrenceOverrides.length, fakeOccurrenceOverrides),`

Reply:
yeah, and maybe the fakeOccurrences function and other could be improved for the count to always match the array length